tools/gen-intrinsics-specs.py

```diff
@@ -93,6 +107,12 @@ def quote_split_intrinsics(intrinsic, source_syntax):
 
     >>> quote_split_intrinsics('int f(int x)', 'markdown')
     '`int f(int x)`'
+
+    >>> quote_split_intrinsics('int f(int x, float y)', 'pdf')
+    '`int f(`\n\n&nbsp;&nbsp;&nbsp;&nbsp;`int x,`\n\n&nbsp;&nbsp;&nbsp;&nbsp;` float y)`'
```

Comment:
I think this is wrong. Here I was expecting markdown syntax for `c` code

```
` ` `c
int f(
        int x,
        float y)
` ` `
```
(please notice that the 3 ` should not be separated by space)

Essentially, I would expect to the the same output of the rst format for grid table, just with `.. code:` replaced by the thre ` +c, and no indentation